diff --git a/node_modules/eris/lib/structures/Message.js b/node_modules/eris/lib/structures/Message.js
index c3f6a15..5d6cf23 100644
--- a/node_modules/eris/lib/structures/Message.js
+++ b/node_modules/eris/lib/structures/Message.js
@@ -86,20 +86,12 @@ class Message extends Base {
 
     if (data.message_snapshots && this.messageReference) {
       this.messageSnapshots = data.message_snapshots.map((snapshot) => {
-        const channel = this._client.getChannel(this.messageReference.channelID);
-        let message;
-
         snapshot.message.id = this.messageReference.messageID;
-
-        if (channel) {
-          message = channel.messages.update(snapshot.message, this._client);
-        } else {
-          message = new Message(snapshot.message, this._client);
-        }
+        snapshot.message.channel_id = this.messageReference.channelID;
 
         return {
           guildID: this.messageReference.guildID,
-          message: message,
+          message: new Message(snapshot.message, this._client),
         };
       });
     }
@@ -112,8 +104,6 @@ class Message extends Base {
       } else {
         this.author = new User(data.author, client);
       }
-    } else {
-      this._client.emit("error", new Error("MESSAGE_CREATE but no message author:\n" + JSON.stringify(data, null, 2)));
     }
 
     if (data.referenced_message) {
@@ -156,7 +146,7 @@ class Message extends Base {
           data.member.user = data.author;
         }
         this.member = this.channel.guild.members.update(data.member, this.channel.guild);
-      } else if (this.channel.guild.members.has(this.author.id)) {
+      } else if (this.author && this.channel.guild.members.has(this.author.id)) {
         this.member = this.channel.guild.members.get(this.author.id);
       } else {
         this.member = null;
